Edits from @jessegmeyerlab (#1120)
greenelab/covid19-review: content/10.diagnostics.md

A second study [@doi:10.1016/j.talanta.2020.121726] confirmed that RT-ddPCR is able to detect SARS-CoV-2 at a lower threshold for viral load relative to RT-PCR.
This study analyzed 196 samples, including 103 samples from suspected patients, 77 from contacts and close contacts, and 16 from suspected convalescents, using both RT-qPCR and RT-ddPCR.
Of the 103 suspected patient samples, RT-qPCR identified 29 as positive, 25 as negative, and 49 as suspected.
Of the RT-qPCR negative or suspected samples, a total of 61 (19 negative and 42 suspected) were confirmed to be positive by RT-ddPCR.
All 103 of the suspected patients were later confirmed to be SARS-CoV-2 positive through a combination of symptom development and RT-qPCR resampling, indicating that RT-ddPCR improved the overall detection rate among these patients from 28.2% to 87.4%.
Of 77 patient samples from contacts and close contacts, 48 tested negative with both methods, and these patients were observed to remain healthy over a period of 14 days.

> Within the remaining 29 patient samples, 12 tested positive, 1 negative, and 16 suspected with RT-qPCR.<!--What does suspected mean hear?-->

Author: ```suggestion
Within the remaining 29 patient samples, 12 were identified as positive, 1 as negative, and 16 as borderline with RT-qPCR.
```

Additionally, PCR-based tests can remain positive for a much longer time than the virus is likely to be actively replicating [@doi:10.1056/NEJMcp2117115], raising concerns about their informativeness after the acute phase of the disease.
Hence the CDC has advised individuals who suspect they have been re-infected with SARS-CoV-2 to avoid using diagnostic tests within 90 days of receiving a previous positive test [@url:https://www.cdc.gov/coronavirus/2019-ncov/your-health/quarantine-isolation.html].

Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
For example, viral shedding kinetics are still not well understood but are expected to introduce a significant effect of timing of sample collection on test results [@doi:10.1038/d41587-020-00002-2].
Similarly, the type of specimen could also influence outcomes, as success in viral detection varies among clinical sample types [@doi:10.1038/d41587-020-00002-2; @doi:10/gjrfns; @doi:10.2807/1560-7917.ES.2020.25.50.2000568].

> With CRISPR-based testing strategies, the gRNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.<!--Clarification: I don’t think this should be “patient’s genome” because the virus doesn’t integrate into human genomes?-->

Author: @SystemsResearch @nilswellhausen would either of you be able to confirm whether this line is what was intended?

Author: @jessegmeyerlab I found a reference that I think agrees with your concern. I propose this change:

```suggestion
With CRISPR-based testing strategies, the gRNA can recognize off-target interspersed sequences in the viral genome [@doi:10.3389/fcimb.2020.576875], potentially resulting in false positives and a loss of specificity.
```

Reviewer: That makes more sense thank you!

Although several molecular diagnostic tests to detect viral genetic material have high selectivity and sensitivity, they provide information only about active infection, and therefore offer just a snapshot-in-time perspective on the spread of a disease.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In such contexts, serological tests are informative.
Serological tests use many of the same technologies as the immunoassays used to detect the presence of an antigen, but are instead used to evaluate the presence of antibodies against SARS-CoV-2 in a serum sample.
These tests are particularly useful for insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
Immunoassays can detect antibodies produced by the adaptive immune system in response to viral threat.
Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.

> The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.<!--Needs Citation-->

Author: Book is also added as a manual ref under URL, so see if this works. If it does, can remove.

```suggestion
The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@doi:10.3389/fimmu.2014.00520; @isbn:9780815336426].
```

> The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@doi:10.3389/fimmu.2014.00520; @isbn:9780815336426].

Author: We probably want to use the manual reference :laughing:![](https://user-images.githubusercontent.com/8975841/164080735-6e030361-71a7-4c4b-8021-57c7cbffd949.png)

![](https://user-images.githubusercontent.com/8975841/164080713-74990980-3d1f-4319-9b47-d5b9fa7eaf6d.png)

Author: ```suggestion
The two immunoglobulin classes that are most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies [@url:https://www.ncbi.nlm.nih.gov/books/NBK27162].
```

Such tests can also illuminate the progression of viral disease, as IgM are the first antibodies produced by the body and indicate that the infection is active.
Once the body has responded to the infection, IgG are produced and gradually replace IgM, indicating that the body has developed immunogenic memory [@doi:10.1002/jmv.25820].
Therefore, it was hoped that the development of assays to detect the presence of IgM and IgG antibodies against SARS-CoV-2 would allow the identification of cases from early in the infection course (IgM) and for months or years afterwards (IgG).
Several technologies have been used to developed serological tests for COVID-19, including ELISA, lateral flow immunoassay, chemiluminescence immunoassay, and neutralizing antibody assays [@doi:10.1093/ajcp/aqaa112].

#### ELISA

The application of ELISA to serological testing is complementary to its use in molecular diagnostics (described above).
Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, the probe is an antigen and the target is an antibody.

> The enzyme used for detection and signal amplification is on a secondary antibody raised generally against human IgG or IgM.<!--Request review on this sentence-->

Author: @cbrueffer @nilswellhausen @RLordan Is anyone able to review this sentence? (review requested by the author, @jessegmeyerlab)

Reviewer 2: I think this is correct + anti-human pan Ig as per: https://doi.org/10.1101/2020.04.24.057323

For instance, a study from Imperial College estimates that in Italy, the true number of infections was around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28, 2020 [@doi:10.25561/77731].
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
Technological advancements that facilitate widespread, rapid testing would therefore be valuable for accurately assessing the rate of infection and aid in controlling the virus' spread.
Additionally, the trade off of accessibility, sensitivity, and time to results has raised some complex questions around which tests are best suited to certain situations.

> Immunoassays, including serological tests, have much lower limits of detection than PCR tests do [@doi:10.2142/biophysico.bppb-v18.004].<!-- check accuracy, this sentence is likely flipped?-->

Author: I wrote this and I definitely flipped the order!

```suggestion
Immunoassays, including serological tests, have much higher limits of detection than PCR tests do [@doi:10.2142/biophysico.bppb-v18.004].
```
